test/conformance/conformance_test.go

```diff
@@ -47,13 +47,19 @@ func TestGatewayAPIConformance(t *testing.T) {
 		validUniqueListenerPorts = []v1alpha2.PortNumber{}
 	}
 
+	supportedFeatures := suite.AllFeatures
+	// ignore mesh conformance tests
+	for feature := range suite.MeshCoreFeatures {
```

Comment:
instead of doing this manually, you want to leverage the `ExemptFeatures` field which handles this for us